src/services/feature-flags/FeatureFlagsService.ts

```diff
@@ -79,6 +79,7 @@ export class FeatureFlagsService {
 	 * Convenience: multi-root workspace remote gate
 	 */
 	public getMultiRootEnabled(): boolean {
+		// return true  // For testing
```

Comment:
```suggestion
```

Reply:
Just left it so that you folks could test and I will remove before merge.